src/main/java/com/springBoot/project/daoService/ClientDaoService.java

```diff
@@ -0,0 +1,153 @@
+package com.springBoot.project.daoService;
+
+import java.sql.CallableStatement;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.log4j.Logger;
+import org.springframework.stereotype.Repository;
+
+import com.springBoot.project.entity.Client;
+import com.springBoot.project.mysqlConnect.ConnectionManager;
+
+@Repository
+public class ClientDaoService implements IClientDaoService {
+	private Connection connection = ConnectionManager.getConnection();
+
+	private static final Logger logger = Logger.getLogger(ClientDaoService.class);
+	
+	private static String QUERY_GET_ALL_CLIENTS = "{SELECT id, lastName, name, numberPassport, address, phone FROM `client`}";
+	private static String QUERY_SELECT_CLIENT_BY_ID = "{SELECT id, lastName, name, numberPassport, address, phone  FROM `client` WHERE id= :id}";
+	private static String QUERY_CREATE_CLIENT = "{INSERT INTO `client`(lastName, name, numberPassport, address, phone) VALUES(:Sname, :name, :passport, :homeaddress, :telephone)}";
+	private static String QUERY_UPDATE_CLIENT_BY_ID = "{UPDATE `client` SET lastName= :lastName, name= :name, numberPassport= :numberPassport, address= :address, phone= :phone WHERE id= :id}";
+	private static String QUERY_DELETE_CLIENT = "{DELETE FROM `client` WHERE id= :id}";
+	
+	/**
+	 * This method select all client form DB
+	 * @return List<Client>
+	 */
+	@Override
+	public List<Client> getAll(){	
+		try
+		{
+			CallableStatement statement = connection.prepareCall(QUERY_GET_ALL_CLIENTS);
```

Comment:
Do you create stored procedures for your queries in the database and call them in your program? If no then change `CallableStatement`->`PreparedStatement` and `.prepareCall`->`.prepareStatement`
For additional information refer to: https://www.tutorialspoint.com/jdbc/jdbc-statements.htm